chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -40,11 +40,13 @@
     mac_list = []
     unless mnode["network"].nil? || mnode["network"]["interfaces"].nil?
       mnode["network"]["interfaces"].each do |net, net_data|
-        net_data.each do |field, field_data|
-          next if field != "addresses"
-          field_data.each do |addr, addr_data|
-            next if addr_data["family"] != "lladdr"
-            mac_list << addr unless mac_list.include? addr
+        if !net.start_with?('tap', 'qvo', 'qbr', 'qvb', 'ovs')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.